src/frontends/tensorflow/src/decoder_argdef.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "decoder_argdef.hpp"
+
+#include "op_def.pb.h"
+#include "openvino/frontend/tensorflow/node_context.hpp"
+#include "openvino/frontend/tensorflow/special_types.hpp"
+#include "types.pb.h"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+
+namespace {
+const std::map<::tensorflow::DataType, ov::element::Type>& TYPE_MAP() {
```

Comment:
I would like to leave Decoder implementation to be standalone to avoid any conflicts with Protobuf versions, you should remember. OVTF can forget or can happen something different that this TYPE_MAP will be taken for other protobuf version.
It is a bit risky to place TYPE_MAP in different file.